documentation/airport-guides/ltba.md

```diff
@@ -0,0 +1,123 @@
+### About Istanbul Atatürk Airport (IST | LTBA)
+*Istanbul Atatürk Airport* is the main airport of Turkey. It serves as a hub for *AtlasGlobal*, *Onur Air* and, most importantly, *Turkish Airlines*, which has the biggest share at the airport and serves numerous destinations worldwide. The airport is also used for cargo operations by cargo airlines such as *Turkish Airlines Cargo*, *MNG Airlines*, *MyCargo Airlines* and *Qatar Airways Cargo*.
+The airport was originally named *Yeşilköy Airport*. In the 1980s, it was renamed *Istanbul Atatürk International Airport* in honour of *Mustafa Kemal Atatürk*, the founder and first president of the Republic of Turkey. It served more than 60 million passengers in 2015, making it the 11th-busiest airport in the world in terms of total passenger traffic and the 10th-busiest in the world in terms of international passenger traffic. As of 2017, it is Europe's 5th-busiest airport after *London Heathrow*, *Paris Charles de Gaulle*, *Frankfurt Airport* and *Amsterdam Schiphol Airport*, having fallen from third place after a decline in passengers due to security fears.
+
+#### Statistics
+* **Total passengers**: *63,727,448* (2017)
+* **International passengers**: *44,277,101* (2017)
+* **Aircraft movements**: *460,777* (2017)
+* **Cargo**: *790,744 tonnes* (2017)
+
+
+### Airspace
+The airspace of the Yeşilköy Approach/Radar Sector encompasses a large area around Istanbul up to FL240. It extends up to 80nm from the airport.  
+The Yeşilköy Approach/Radar Sector also controls arrivals and departures of Istanbul's secondary airport *Sabiha Gökçen International Airport (SAW | LTFJ)*.
+
+
+### Runways
+The airport has 3 runways. The southeastern runway (05/23) is 2580m (8465ft) long. The other two runways (17R/35L; 17L/35R) are parallel but too close for independent simultaneous approaches. *Atatürk Airport* applies a *"Preferential Runway System"* (PRS) indicating the runway or runways that are considered by ATC to be the most suitable for take-off or landing, taking aircraft performance, surface wind speed and other components into consideration. During *PRS* operations the following runway configuration is usually used:
+
+* Runway 05 for landing
+* Runway 35L for take-off
+
+**This is also the configuration in use in openScope**
+
+
+### Instrument Procedures
+*Istanbul Atatürk Airport* makes use of a large number of SIDs and STARs, both RNAV and non-RNAV, although only RNAV procedures are normally used.
```

Comment:
Rewording:
`*Istanbul Atatürk Airport* has a large quantity of SIDs and STARs, RNAV and non-RNAV. However, only the RNAV procedures are normally used.`